MvvmCross.Forms/Presenters/Attributes/MvxModalPresentationAttribute.cs

```diff
@@ -12,5 +12,38 @@ public class MvxModalPresentationAttribute : MvxPagePresentationAttribute
         public MvxModalPresentationAttribute()
         {
         }
+
+        /// <summary>
+        /// set the modal presentation style of the modal. This is used for iOS devices for setting different types of modals.
+        /// </summary>
+        public ModalPresentationStyle PresentationStyle { get; set; }
+    }
+
+    public enum ModalPresentationStyle
```

Comment:
This should probably be prefixed with `Mvx` like other MvvmCross types. Maybe even more specific with `MvxForms`:

```suggestion
    public enum MvxFormsModalPresentationStyle
```

Reply:
Renamed to `MvxFormsModalPresentationStyle`